package-lock.json

```diff
@@ -22227,4 +22227,4 @@
       }
     }
   }
-}
+}
```

Comment:
Annoying change but can you add a new line at the end?

Reply:
sure